views/projects.tpl

```diff
@@ -60,6 +60,12 @@
                 }
             })
 
+            // handle old JSON encoding of query
+            const old_json_matches = decodeURIComponent(window.location.hash)
+                .match(/^#{"dropdown":"([^"]*)","input":"([^"]*)"}$/);
```

Comment:
Did you try if this works?

Reply:
Yeah, of course, it doesn't on your side?

Reply:
Sorry I doubted you...